packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -142,6 +145,8 @@ export default function Sidebar({
   const { sidebarExpanded, toggleSidebarExpanded, loadedSettings } =
     useContext(SettingsContext);
   const [showSettings, setShowSettings] = useState(false);
+  const { flags } = useContext(FeaturesContext);
+  const shouldShowMyFeed = getFeatureValue(Features.MyFeedOn, flags, 'false');
```

Comment:
This is quite confusing because usually if you prefix a variable name with `should` you expect it to be boolean whereas in this case, it's a string.
Also from a Flagsmith point of view, we can utilize the `enabled` field instead of the actual value. So for the person who manages the deployment (aka Nimrod), it will be more straightforward and less error-prone.
What do you think?

Reply:
Agreed, let me rework the `getFeatureValue` to handle booleans :)